spanner/spanner_snippets/spanner/spanner_batch_write.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_batch_write]
```

Comment:
I am worried about the sample tag that is used here.
We used `spanner_batch_write_at_least_once` for Java, and this should be consistent across the client libraries as this tag is mapped in the public documentation. We cannot have it as `spanner_batch_write` and most importantly `spanner_batch_write_at_least_once` tag does not properly suit.
Any idea why did we suffix it `with at_least_once`?

Reply:
The Java library has `write()` and `writeAtLeastOnce()` methods for `Commit`, whereas the golang library has a single method `Apply()` with `ApplyAtLeaseOnce()` as an option.

Since the current batch write API is non-replay protected, it is named `batchWriteAtLeastOnce()` in Java, and the hence the region tag is named such. In golang, we'd have an option in `BatchWriteOptions` instead of a new method.

While it is technically feasible to have different region tags, it is not ideal as you mentioned. Updated this PR to use the same region tag here as well.